inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convert_like.hpp

```diff
@@ -24,13 +24,15 @@ using ConvertLikeParamsTuple = typename std::tuple<
         InferenceEngine::Layout,           // Output layout
         std::string>;                      // Device name
 
-class ConvertLikeLayerTest : public testing::WithParamInterface<ConvertLikeParamsTuple>,
-                             virtual public LayerTestsUtils::LayerTestsCommon {
+class NGRAPH_DEPRECATED("This class is deprecated and will be removed soon. "
```

Comment:
I think we can change test classes without deprecation.
Moreover it is Inference Engine tests, I think will be better to use deprecation macros from Inference Engine instead of nGraph.

Reply:
I updated the code to use Inference Engine macros.